modules/apps/portal/portal-user-locale-options-web/src/main/resources/META-INF/resources/dynamic_include/user_locale_options.jsp

```diff
@@ -22,7 +36,7 @@ Locale userLocale = user.getLocale();
 		<c:if test="<%= LanguageUtil.isAvailableLocale(themeDisplay.getSiteGroupId(), user.getLocale()) || (PortalUtil.isGroupControlPanelPath(themeDisplay.getURLCurrent()) && LanguageUtil.isAvailableLocale(userLocale)) %>">
 			<clay:link
 				cssClass="d-block"
-				href='<%= themeDisplay.getPathMain() + "/portal/update_language?redirect=" + URLCodec.encodeURL(themeDisplay.getURLCurrent()) + "&groupId=" + themeDisplay.getScopeGroupId() + "&privateLayout=" + layout.isPrivateLayout() + "&layoutId=" + layout.getLayoutId() + "&languageId=" + user.getLanguageId() + "&persistState=false&showUserLocaleOptionsMessage=false" %>'
+				href='<%= themeDisplay.getPathMain() + "/portal/update_language?redirect=" + URLCodec.encodeURL(themeDisplay.getURLCurrent()) + "&groupId=" + groupId + "&privateLayout=" + privateLayout + "&layoutId=" + layoutId + "&languageId=" + user.getLanguageId() + "&persistState=false&showUserLocaleOptionsMessage=false" %>'
```

Comment:
can we sort the parameters to be groupId / layoutId / languageId / persistState / privateLayout / ... ?

Reply:
The last time we tried to sort it in that manner, BChan changed it, see here: https://github.com/liferay/liferay-portal/commit/501f884c6e71b3d68a68b8c8ad0d1497f16c2e6c